fastlane_bot/bot.py

```diff
@@ -368,7 +368,13 @@ def _run(
             if logging_path:
                 filename = f"tx_{datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}.txt"
                 with open(os.path.join(logging_path, filename), "w") as f:
-                    f.write(f"{tx_hash} {tx_status}: {tx_details}")
+                            log_entry = {
+                                "tx_hash": tx_hash,
+                                "tx_status": tx_status,
+                                "tx_details": tx_details,
+                                "log_dict": log_dict
+                            }
+                            json.dump(log_entry, f, indent=4)
```

Comment:
@barakman changed this to a json dump.
The tx_details one looks a little odd but it is functional